mock-server/server.py

```
@@ -0,0 +1,463 @@
+            self._send_empty(HTTPStatus.OK)
+            return
+
+        self._not_found()
+
+    def log_message(self, fmt: str, *args: Any) -> None:
+        print(f"[{self.log_date_time_string()}] {self.client_address[0]} {fmt % args}")
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Run local mock backend for android-shopping-order")
+    parser.add_argument("--host", default="0.0.0.0", help="Bind host")
```

Ruff (0.15.13)

[error] Possible binding to all interfaces

(S104)